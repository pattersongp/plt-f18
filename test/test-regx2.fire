func int main = () => {
    regx anything = ".*";
    
    if(anything === "match"){
        sprint("match1");
    }

   if(anything === "MATCH"){
       sprint("match2");
    }

    if(anything === "1234") {
        sprint("match3");
    }

    if(anything === ""){
        sprint("match4");
    }
}
