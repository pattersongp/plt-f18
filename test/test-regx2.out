match1
match2
match3
match4
